### DISCUSSION TITLE
Nuget.org reserved prefixes

### DISCUSSION THREAD

**Discussion by ismaelhamed** [akkadotnet/akka.net#6790] [General]
@Aaronontheweb I see you guys reserved all Akka.* prefixes in nuget.org. Since future community's extensions to the project will now need to be published under a different prefix, IMO that will definitely hurt discoverability of those packages. Have you guy consider making a subset (i.e., Akka.Contrib.*) public?

**Comment by Aaronontheweb** [comment]
I'm not convinced that it hurts or helps discoverability - I.e. Microsoft reserving ASP.NET's prefixes hasn't stopped stop users from creating, discovering, or finding third party plugins, including some very popular ones. 

We reserved the prefix so we have control over which ones look like they come from our project or not.

You can still use the Akka name in new packages - just not Akka.{PackageName}. I can see about releasing the Akka.Contrib namespace for third party use - I'm not opposed to that, just need to see if NuGet supports having an exception to the Akka.* prefix.

**Comment by ismaelhamed** [comment]
There's still a lot of packages in the JVM that hasn't been ported. Maybe reserving just the package names for the ones under control of the AkkaDotNet team would be more sensible, though I'm not even sure that's an option. GPG verification seems like a better way to check package authenticity.
